src/models/multiChart.js

```diff
@@ -95,7 +95,7 @@ nv.models.multiChart = function() {
                     })
                 });
 
-            x   .domain(d3.extent(d3.merge(series1.concat(series2)), function(d) { return getX(d) }))
+            x   .domain(d3.extent(d3.merge(series1.concat(series2)), function(d) { return d.x }))
```

Comment:
Becaouse this code is so much messed up. Look at this this value is taked before using getX and it is stored in x property.
If you will store x value in other than x property, actual code will never work correctly.

I am actual fixing this nvd3 in our internal project in corporation where I work.
I can easily point much, much more errors in nvd3